tests/integration/framework/process/daprd/options.go

```diff
@@ -136,6 +136,21 @@ func WithResourceFiles(files ...string) Option {
 	}
 }
 
+// WithInMemoryStore adds an in-memory state store component, which is also enabled as actor state store.
+func WithInMemoryStore(storeName string) Option {
```

Comment:
Sorry, my fault when I suggested the name. Can we include `ActorState` to be explicit?

```suggestion
func WithInMemoryActorStateStore(storeName string) Option {
```